pulp_smash/tests/pulp3/file/api_v3/test_repo_version.py

```diff
@@ -0,0 +1,98 @@
+# coding=utf-8
+"""Tests related to repository version."""
+import unittest
+from random import choice
+from urllib.parse import urljoin
+
+from pulp_smash import api, config, utils
+from pulp_smash.constants import FILE_FEED_URL
+from pulp_smash.tests.pulp3.constants import (
+    FILE_CONTENT,
+    FILE_IMPORTER_PATH,
+    FILE_PUBLISHER_PATH,
+    REPO_PATH,
+)
+from pulp_smash.tests.pulp3.file.api_v3.utils import (
+    gen_importer,
+    gen_publisher,
+)
+from pulp_smash.tests.pulp3.file.utils import set_up_module as setUpModule  # noqa pylint:disable=unused-import
+from pulp_smash.tests.pulp3.pulpcore.utils import gen_repo
+from pulp_smash.tests.pulp3.utils import (
+    get_auth,
+    get_latest_repo_version,
+    read_repo_content,
+    sync_repo,
+)
+
+
+class AddingRemovingUnitsTestCase(unittest.TestCase, utils.SmokeTest):
+    """Create new repo version adding or removing content."""
+
+    def test_all(self):
+        """Create a new repo version adding or removing content.
+
+        This test explores the design choice stated in `Pulp #3234`_ that
+        create a new repository version by adding or removing content to the
+        latest version of the repository.
+
+        .. _Pulp #3234: https://pulp.plan.io/issues/3234
+
+        Do the following:
+
+        1. Add content to Pulp - using sync to a first repository.
+        2. Create a second repository.
+        3. Use `add_content_units` to pass an url for a unit to be added
+           to second repository.
+        4. Assert that the repository version has changed for the second
+           repository.
+        5. Assert that there is just one content unit in the second repository.
+        6. Remove the just added unit from the second repository using the
+           `remove_content_units`, and assert that the repository version has
```

Comment:
Single backticks denote inline code in markdown. Double backticks denote inline code reStructuredText.